### DISCUSSION TITLE
How much GPU memory does Whipser large-v3 fine-tuning require?

### DISCUSSION THREAD

**Discussion by toanhuynhnguyen** [SYSTRAN/faster-whisper#1070] [Q&A] [answered]
Hi everyone,
I follow this document: https://huggingface.co/blog/fine-tune-whisper
How much GPU memory does Whipser large-v3 fine-tuning require?
Thank you.

**Comment by MahmoudAshraf97** [comment, marked as answer]
Hello, faster-whisper is  focused on inference only, please check other libraries that support training such as `transformers`

**Reply by toanhuynhnguyen** [reply]
Yes, I follow this guide https://huggingface.co/blog/fine-tune-whisper then I convert the fine-tuned model to CTranslate2 model as this guide https://github.com/SYSTRAN/faster-whisper?tab=readme-ov-file#model-conversion
